.github/workflows/integration-testing.yml

```diff
@@ -0,0 +1,99 @@
+name: Plio Integration Tests
+
+on:
+  pull_request:
+  push:
+    branches: [master]
+
+jobs:
+  cypress:
+    runs-on: ubuntu-latest
+
+    # uses GitHub environment
+    environment:
+      name: Integration Testing
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Plio Frontend
+        run: |
+          # clone the project
+          mkdir -p projects/ && cd projects/
+          git clone https://github.com/avantifellows/plio-frontend/
+          cd plio-frontend/
+
+          # temporary git checkout -- remove before merge
+          git checkout feature/cypress-ci && git pull origin feature/cypress-ci
+
+          # create the env file
+          cp .env.example .env
+
+          # add env secrets
+          echo 'VUE_APP_I18N_LOCALE=${{ secrets.VUE_APP_I18N_LOCALE }}' >> .env
```

Comment:
the `.env` will have two entries for VUE_APP_I18N_LOCALE, including the one from `.env.example`, right?

Reply:
Yes, I think we can remove this as it already has a default value. I'll remove the need of adding it in the github action.